binary_tree.py

```diff
@@ -0,0 +1,56 @@
+'''
+Created on 22 May 2017
+
+@author: Eva
+'''
+
+class BinaryTree:
+    
+    def __init__(self, root=None):
+        self.root = root
+    
+    def get_root(self):
+        return self.root
+    
+    def insert(self, new_node, parent, to_left):
```

Comment:
Additional comment: you could also simply have only the `TreeNode` class and remove `BinaryTree`. In that case, you will have that insert takes the root `TreeNode` and returns the new root. Then a `None` value represents the empty tree.